Source/WebCore/rendering/RenderLayerModelObject.cpp

```diff
@@ -579,6 +579,58 @@ RenderSVGResourcePaintServer* RenderLayerModelObject::svgStrokePaintServerResour
 
     return nullptr;
 }
+
+bool RenderLayerModelObject::pointInSVGClippingArea(const FloatPoint& point) const
```

Comment:
SVGRenderSupport::pointInClippingArea() explicitly contains cycle detection code - we'd need to add LBSE-only code into SVGRenderSupport to account for that. Since we don't want to add new stuff to mostly-legacy-by-now SVGRenderSupport, the common choice so far was to add stuff to RenderLayerModelObject with a 'SVG' string in the method name - as for applySVGTransform, etc.

That's the reason why this change is included in this PR, it does belong to the cycle detection refactoring.